doc/user-guide/usage.md

```diff
@@ -0,0 +1,380 @@
+  are created by LVMO
+- LVMVolumeGroup: CR created and managed by LVMO which tracks individual volume
+  groups across multiple nodes
+``` json
+# kubectl get lvmvolumegroup test -ojson
+{
+    "apiVersion": "lvm.topolvm.io/v1alpha1",
+    "kind": "LVMVolumeGroup",
+    "metadata": {
+        "creationTimestamp": "2022-02-02T05:16:42Z",
+        "generation": 1,
+        "name": "test",
+        "namespace": "lvm-operator-system",
+        "resourceVersion": "17242461",
+        "uid": "88e8ad7d-1544-41fb-9a8e-12b1a66ab157"
+    },
+    "spec": {}
+}
+```
+- LVMVolumeGroupNodeStatus: CR created and managed by VG Manager which tracks
+  node status corresponding to multiple volume groups
+``` json
+# kubectl get lvmvolumegroupnodestatuses.lvm.topolvm.io kube-node -ojson
+{
+    "apiVersion": "lvm.topolvm.io/v1alpha1",
+    "kind": "LVMVolumeGroupNodeStatus",
+    "metadata": {
+        "creationTimestamp": "2022-02-02T05:17:59Z",
+        "generation": 1,
+        "name": "kube-node",
+        "namespace": "lvm-operator-system",
+        "resourceVersion": "17242882",
+        "uid": "292de9bb-3a9b-4ee8-946a-9b587986dafd"
+    },
+    "spec": {
+        "nodeStatus": [
+            {
+                "devices": [
+                    "/dev/nvme0n1",
+                    "/dev/nvme1n1",
+                    "/dev/nvme2n1"
+                ],
+                "name": "test",
+                "status": "Ready"
+            }
+        ]
+    }
+}
+```
+- A storage class which matches names (prefixed with `topolvm-`) of
+  deviceClasses in LVMCluster CR are created as below
+``` console
+# kubectl get storageclass
+NAME          PROVISIONER          RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
+topolvm-vg1   topolvm.cybozu.com   Delete          WaitForFirstConsumer   true                   31m
+```
+
+Note:
+- Reconciling multiple LVMCluster CRs by LVMO is not supported
+- Custom resources LVMVolumeGroup and LVMVolumeGroupNodeStatus are managed by
+  LVMO and users should not edit them.
+
+## Deploying PVC and App Pod
+
+- A successful reconciliation of LVMCluster CR will setup all the underlying
+  resources needed to be able to create a PVC and use it in app pod
+- Create a PVC using the StorageClass created for the deviceClass. The PVC will
+  not be bound until a pod claims the storage as the volume binding mode is set
+  to WaitForFirstConsumer.
+```yaml
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: lvm-file-pvc
+spec:
+  volumeMode: Filesystem
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 5Gi
+  storageClassName: topolvm-vg1
+EOF
+```
+``` console
+kubectl get pvc
+NAME           STATUS    VOLUME   CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+lvm-file-pvc   Pending                                      topolvm-vg1    12s
+```
+- In a multi node setup this volume binding is used to take scheduling
```

Comment:
Which "volume binding" is this referring to? I don't see how the scheduler prefers nodes with highest available storage space.

Reply:
- volume binding is just mentioned in previous bullet point, i.e, WaitForFirstConsumer

> I don't see how the scheduler prefers nodes with highest available storage space
- IMHO, this detail isn't needed in user guide
- or at the least I can link to the k8s blog about csistorage capacity tracking then, not sure though